components/src/dynamo/trtllm/request_handlers/diffusion/__init__.py

```
@@ -6,8 +6,11 @@
 """
 
-from dynamo.trtllm.request_handlers.video_diffusion.video_handler import (
+from dynamo.trtllm.request_handlers.diffusion.image_handler import (
+    ImageGenerationHandler,
+)
+from dynamo.trtllm.request_handlers.diffusion.video_handler import (
     VideoGenerationHandler,
 )
 
-__all__ = ["VideoGenerationHandler"]
+__all__ = ["VideoGenerationHandler", "ImageGenerationHandler"]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)